src/site/content/en/blog/sms-receiver-announce/index.md

```diff
@@ -0,0 +1,401 @@
+<code>PqEvUq15HeK</code>
+</td>
+</tr>
+<tr>
+<td markdown="block">
+Chrome Dev
+</td>
+<td markdown="block">
+
+</td>
+</tr>
+<tr>
+<td markdown="block">
+Chrome Beta
+</td>
+<td markdown="block">
+<code>xFJnfg75+8v</code>
+</td>
+</tr>
+<tr>
+<td markdown="block">
+Chrome Stable
+</td>
+<td markdown="block">
+<code>EvsSSj4C6vl</code>
+</td>
+</tr>
+<tr>
+<td markdown="block">
+Locally built Chromium
+</td>
+<td markdown="block">
+<code>s3LhKBB0M33</code>
+</td>
+</tr>
+</table>
+
+### Demos
+
+Try various messages with the demo:
+[https://sms-receiver-demo.glitch.me](https://sms-receiver-demo.glitch.me)
+
+You may also fork it and create your version:
+[https://glitch.com/edit/\#!/sms-receiver-demo](https://glitch.com/edit/#!/sms-receiver-demo).
+
+### Enabling support during the origin trial
+
+Starting in Chrome 78, the SMS Receiver API will be available as an origin trial
+on Chrome for Android. Origin trials allow you to try new features and give
+feedback on their usability, practicality, and effectiveness, both to us, and to
+the web standards community. For more information, see the [Origin Trials Guide
+for Web
+Developers](https://googlechrome.github.io/OriginTrials/developer-guide.html).
+
+To participate in an origin trial:
+
+1. Request a [token](https://developers.chrome.com/origintrials/) for your
+   origin.
+1. Add the token to your pages, there are two ways to provide this token on any
+   pages in your origin
+    * Add an origin-trial `<meta>` tag to the head of any page. For example,
+      this may look something like: `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
+    * If you can configure your server, you can also provide the token on pages
+      using an `Origin-Trial` HTTP header. The resulting response header should
+      look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+## Feedback
+
+We want to hear about your experiences with the SMS Receiver API.
+
+### Tell us about the API design
+
+Is there something about the API that doesn't work as expected? Or are there
+missing methods or properties that you need to implement your idea?
+
+* File an issue on the [SMS Receiver API explainer GitHub
+  repo](https://github.com/samuelgoto/sms-receiver), or add your thoughts to an
+  existing issue.
+
+### Problem with the implementation?
+
+Did you find a bug with Chrome's implementation? Or is the implementation
+different from the spec?
+
+* File a bug at
+  [https://new.crbug.com](https://bugs.chromium.org/p/chromium/issues/entry?components=Blink%3EContacts).
+  Include as much detail as you can, simple instructions for reproducing, and
+  set **Components** to Blink&gt;SMS.
+
+### Planning to use the API?
+
+Planning to use the SMS Receiver API? Your public support helps us prioritize
+features, and shows other browser vendors how critical it is to support them.
+
+* Send a Tweet to [@ChromiumDev](https://twitter.com/chromiumdev) with
+  \#smsreceiver and let us know where and how you're using it.
```

Comment:
Can you wrap `#smsreceiver` in italics, or code font or something so it sticks out a bit, please?